pkg/network/msg.go

```diff
@@ -59,3 +59,15 @@ type DataRequest struct {
 	ID        string `json:"id"`
 	SensorIds []int  `json:"sensorIds"`
 }
+
+// DataUpdate represents the incoming update data command
+type DataUpdate struct {
+	ID   string `json:"id"`
+	Data []Data `json:"data"`
+}
+
+// Data represents the thing's data
+type Data struct {
```

Comment:
I think it'd better move this structure to the `entities` folder. It's a domain construction. Take a look at [`SchemaUpdateRequest`](https://github.com/CESARBR/knot-babeltower/pull/50/commits/a585a844b7f491ea08937ca2c4b79ff4de29cb65#diff-bb05fc85fdd18a712efa5c6b38a24032R30).

Reply:
@joaoaneto You are right! Done!